Jenkinsfile

```diff
@@ -490,8 +505,23 @@ pipeline {
             description: "Enable OpenCL backend stages")
         booleanParam(
             name: "PERF_TEST",
-            defaultValue: false,
+            defaultValue: true,
```

Comment:
@JehandadKhan  should i reset this to false?

Reply:
Yes, please